src/utils/response.ts

```diff
@@ -25,7 +25,9 @@ export function defaultContentType (event: CompatibilityEvent, type?: string) {
 export function sendRedirect (event: CompatibilityEvent, location: string, code = 302) {
   event.res.statusCode = code
   event.res.setHeader('Location', location)
-  return send(event, 'Redirecting to ' + location, MIMES.html)
+  // minimal html document that redirects on client side
+  const html = `<!DOCTYPE html><html><head><meta http-equiv="refresh" content="0; url=${encodeURI(location)}"></head><body></body></html>`
```

Comment:
Can you please also add a visual `<a>` link?

Reply:
I've added it within a `<noscript>` as otherwise I think it is a better experience not to have text flash on the screen and then off again. Let me know if you feel differently.

Reply:
I see but with browser fallback, there is anyway a flash to white...